utils/screen.js

```diff
@@ -0,0 +1,10 @@
+import { getWindowSize } from "../src/components/WindowSize";
+
+/**
+ * Because Next.js does SSR, it does not have access to Window and Document APIs.
+ * NOTE: any component that calls this method should be dynamically imported.
+ * This will allow it to capture the client's browser data
+ */
+export const displayMobileView = () => {
```

Comment:
Just a thought, should we simply let the frontend components using this function pass the width?
If they need to use this function, they should use componentDidMount to get the window property on client side and pass that information to this function.